lib/order.rb

```diff
@@ -0,0 +1,36 @@
+require './lib/send_sms'
+
+class Order
+
+  attr_reader :menu
+
+  def initialize
+    @order = []
+  end
+
+  def add_menu(menu)
+    @menu = menu
+  end
+
+  def add_dish(dish)
+    @order.append(dish.name => dish.price)
+  end
+
+  def contents
+    @order
+  end
+
+  def total
+    total = 0
+    @order.each do |x|
+      total += x.values.sum
+    end
+    total
+  end
+
+  def place_order
+    sms = SMS.new
```

Comment:
The `SMS` instance could be dependency-injected in this class, rather than directly instantiated here  — it'll make it easier to mock in the tests